packages/stark-core/src/modules/http/services/http.service.ts

```diff
@@ -40,22 +40,18 @@ export class StarkHttpServiceImpl<P extends StarkResource> implements StarkHttpS
 		@Inject(STARK_LOGGING_SERVICE) private logger: StarkLoggingService,
 		@Inject(STARK_SESSION_SERVICE) private sessionService: StarkSessionService,
 		private httpClient: HttpClient
-	) {
-		this.logger = logger;
-		this.sessionService = sessionService;
-		this.httpClient = httpClient;
-	}
+	) {}
 
 	public executeSingleItemRequest(request: StarkHttpRequest<P>): Observable<StarkSingleItemResponseWrapper<P>> {
-		// NG-1361: remove the etag before executing the request
+		// remove the etag before executing the request
 		request = this.removeETagFromRequestItem(request);
 
 		// NG-1346: fake pre-authentication support
```

Comment:
This reference to our internal JIRA issue should also be removed... and the "fake pre-authentication" words also :p